dio/pubspec.yaml

```diff
@@ -11,12 +11,11 @@ environment:
 dependencies:
   dio: ^4.0.0
   sentry: ^6.6.1
-  meta: ^1.3.0
 
 dev_dependencies:
+  meta: ^1.3.0
   lints: ^2.0.0
   test: ^1.21.1
-  yaml: ^3.1.0 # needed for version match (code and pubspec)
```

Comment:
Should we rather keep and add the `version_test`?

Reply:
Sure

Reply:
https://github.com/getsentry/sentry-dart/issues/919